### PR TITLE
[MM-54878] Replace usage of LocalizedIcon in 'plain_log_list.tsx' with i/span tags

### PR DESCRIPTION
**Pull request by Sudhanva-Nadiger**
- fixes: #25083

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-54878

#### Release Note

```release-note
None
```

### CODE REVIEW

**Comment by M-ZubairAhmed** [comment]
@Sudhanva-Nadiger amazing work before we can start the review can you please follow guidelines of PR checklist mentioned https://github.com/mattermost/mattermost/issues/25083#issue-1958836587

**Comment by Sudhanva-Nadiger** [comment]
We don't have any snapshot update for this file ! As there is no test for this file ! 

And for the screenshot i couldn't get where this component comes in the app ! If you could help me that would be great

**Comment by M-ZubairAhmed** [comment]
> We don't have any snapshot update for this file ! As there is no test for this file !
> 
> And for the screenshot i couldn't get where this component comes in the app ! If you could help me that would be great

Hey, the way i would do is to first find out where is the component in question being rendered, and next where is that parent rendered, that's how i climb the ladder until i find something significant or a big component which i know where it is in the app.

Hint : this is inside of admin console page

File: webapp/channels/src/components/admin_console/server_logs/plain_log_list.tsx [hunk 1]
```diff
@@ -96,9 +95,9 @@ export default class PlainLogList extends React.PureComponent<Props, State> {
                     className='btn btn-tertiary filter-control filter-control__prev'
                     onClick={this.previousPage}
                 >
-                    <LocalizedIcon
+                    <i
                         className='fa fa-angle-left'
-                        title={{id: t('generic_icons.previous'), defaultMessage: 'Previous Icon'}}
+                        title={this.props.intl.formatMessage({id: t('generic_icons.previous'), defaultMessage: 'Previous Icon'})}
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/admin_console/server_logs/plain_log_list.tsx:100]: t should not be neede here

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Comment by M-ZubairAhmed** [comment]
@Sudhanva-Nadiger gentle ping on this one and i understand you are busy with your other ongoing prs thank you for that 😄

**Review by M-ZubairAhmed — APPROVED** [review]
As always consistent quality contribution 💥💥

**Review by yasserfaraazkhan — APPROVED** [review]
tested this locally by changing config.
working as expected.

**Comment by Sudhanva-Nadiger** [comment]
@M-ZubairAhmed one non related commit was pushed before ! so i had to drop it and force pushed ! sorry for the inconvenience

**Pull request merged**